herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -256,8 +256,12 @@
 
             uoc = unique_objects_container('baseclass', self.stored_baseclass);
             glc = self.global_container('value', self.global_name_);
-            for i=1:self.n_objects
-                uoc = uoc.add( glc{ self.idx_(i) } );
+            for ii=1:self.n_objects
+                obj   = self.get(ii);
+                hash  = self.hash(ii);
```

Comment:
Is this meant to be `hash(obj)`?

Reply:
no, that would be self.global_container(...).hashify(obj). This gets out the existing hash for position ii of the container